corgi/api/serializers.py

```diff
@@ -289,12 +289,16 @@ class SoftwareBuildSerializer(IncludeExcludeFieldsSerializer):
 
     @staticmethod
     def get_link(instance: SoftwareBuild) -> str:
-        return get_model_id_link("builds", instance.build_id)
+        return get_model_id_link("builds", instance.uuid)
```

Comment:
There were some changes to how we build the URLs and provide these links in one of Jim's recent MRs. Because of that, I'm not sure if this get_model_id_link() function still works correctly. If you test this and it works, I don't have any concerns.

We also said we wanted to use persistent identifiers instead of UUIDs, but if you're fine with this approach I don't see a problem. We won't be reloading data anymore so the UUIDs shouldn't change.

Finally, let's consistently use pk so we don't have to keep changing these (and to match below):
```suggestion
        return get_model_id_link("builds", instance.pk)
```

Reply:
Ok, I will update this.